<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAnnotationPictureWidgetBase</class>
 <widget class="QWidget" name="QgsAnnotationPictureWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Text Annotation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QCheckBox" name="mLockAspectRatioCheck">
     <property name="text">
      <string>Lock aspect ratio</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="mBackgroundCheckbox">
     <property name="title">
      <string>Draw background</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Symbol</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsSymbolButton" name="mBackgroundSymbolButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Change…</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="mBorderCheckbox">
     <property name="title">
      <string>Draw border</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Symbol</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsSymbolButton" name="mBorderSymbolButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Change…</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QgsPictureSourceLineEditBase" name="mSourceLineEdit" native="true">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="mRadioRaster">
       <property name="text">
        <string>Raster image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mRadioSVG">
       <property name="text">
        <string>SVG image</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QgsAnnotationItemCommonPropertiesWidget" name="mPropertiesWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsAnnotationItemCommonPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>qgsannotationitemcommonpropertieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsPictureSourceLineEditBase</class>
   <extends>QWidget</extends>
   <header>qgsfilecontentsourcelineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
